lib/datasource/docker/common.ts

```diff
@@ -81,12 +81,20 @@ export async function getAuthHeaders(
         'base64'
       );
       opts.headers = { authorization: `Basic ${auth}` };
+    } else if (opts.token) {
+      const authType = opts.authType ? opts.authType : 'Bearer';
+      logger.debug(
+        `Using ${authType} token for Docker registry ${registryHost}`
+      );
```

Comment:
```suggestion
```

reduce noise

Reply:
Hmmm, here I disagree.
I kept this similar to the case of Basic authentication a few lines below:
```
      logger.debug(`Using Basic auth for docker registry ${registryHost}`);
```

I changed both (for consistency) to `trace`.